### DISCUSSION TITLE
Welcome to flash.nvim Discussions!

### DISCUSSION THREAD

**Discussion by folke** [folke/flash.nvim#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by gsinclair** [comment]
Hi folke, and thanks for this plugin (and many others). I'm enjoying playing with it as I refresh my neovim config for the first time in quite a while, finally based on lazy.nvim.

**Comment by Verisana** [comment]
Hey @folke , thank you very much for all your work you do. That is amazing! :+1: 

I just wanted to ask, if someone knows, how to open more labels in situation like this?

![image](https://github.com/folke/flash.nvim/assets/7406610/440883b7-7548-4e60-9a1c-a95b9d415c1c)

In the beginning I can see labels, but later they are just not enough. In old leap motion, I could cycle through label groups using Tab, but in new version it doesn't work :(
I believe I miss something, but didn't get it yet.

**Reply by Verisana** [reply]
I think I understood it. I should just continue typing next char and it narrows down. Cool!

**Reply by folke** [reply]
Yes exactly :)
